apps/src/applab/designMode.js

```diff
@@ -508,7 +523,9 @@ function makeDraggable (jqueryElements) {
 
     wrapper.css({
       top: elm.css('top'),
-      left: elm.css('left')
+      left: elm.css('left'),
+      width: parseInt(elm[0].style.width, 10) + parseInt(elm[0].style.margin, 10),
+      height: parseInt(elm[0].style.height, 10) + parseInt(elm[0].style.margin, 10)
```

Comment:
It's in base.scss, I believe there so that we naturally have some space between elements. @davidsbailey prob knows most about this. I wouldn't mind getting rid of the margin everywhere, but I suspect this might have some ramifications we don't like.